src/Header/Header.js

```diff
@@ -1,10 +1,12 @@
+import EarthLogo from "../images/earth.svg";
 import "./Header.css";
 
 const Header = () => {
   return (
-    <div>
-      <h1>Header section</h1>
-    </div>
+    <navbar className="navbar">
+      <img className="nav-logo" src={EarthLogo}></img>
+      <p className="nav-text">my travel journal.</p>
+    </navbar>
```

Comment:
we got a typo it's `<nav></nav>` instead of `<navbar></navbar>`
![Screenshot from 2022-11-08 18-48-16](https://user-images.githubusercontent.com/78860665/200638390-89ad9629-0155-4ff3-a6f1-bb49ffb9d313.png)

Reply:
Changed